index.js

```diff
@@ -66,13 +63,12 @@ module.exports = (processor) => {
      */
 
     function jobSpec(opConfig) {
-        _.merge(baseOpConfig, opConfig);
         return {
             'operations': [
                 {
                     '_op': 'noop'
                 },
-                baseOpConfig
+                opConfig
```

Comment:
Doesn't this change the interface of `run()` to now require the `opConfig` argument?

Reply:
Good catch.  Fixed.